kinto/core/initialization.py

```diff
@@ -284,11 +285,19 @@ def setup_sentry(config):
         if env:
             env_options["environment"] = env
 
+        breadcrumbs_level = settings["sentry_breadcrumbs_min_level"]
+        events_level = settings["sentry_events_min_level"]
         sentry_sdk.init(
             dsn,
             integrations=[
                 PyramidIntegration(),
                 SqlalchemyIntegration(),
+                LoggingIntegration(
+                    # Capture debug and above as breadcrumbs
+                    level=breadcrumbs_level,
+                    # Send warnings and above as events
+                    event_level=events_level,
```

Comment:
Nit, but these comments are only true if the default settings are overridden. Should we instead explain what `level` and `event_level` do, and then say "defaults to `<whatever>`"?